src/common/kube-helpers.ts

```diff
@@ -28,28 +28,41 @@ function readResolvedPathSync(filePath: string): string {
   return fse.readFileSync(path.resolve(resolveTilde(filePath)), "utf8");
 }
 
+function checkRawCluster(rawCluster: any): boolean {
+  return rawCluster?.name && rawCluster?.cluster?.server;
+}
+
+function checkRawUser(rawUser: any): boolean {
+  return rawUser?.name;
```

Comment:
Personally, I'm in favor of explicit type conversion here.
```suggestion
  return Boolean(rawUser?.name);
```